### ISSUE TITLE
Imported bluesky timeline (own posts) appears as a duplicate post at x

### ISSUE THREAD

**Issue by ghost** [friendica/friendica#13847]
Imported bluesky timeline appears as a duplicate post at x if the own posts are also imported.
Is it possible that if the "Import the remote timeline" is activated for bluesky, the own posts are imported back to friendica and these then appear again as duplicate posts at "X"?

**Comment by ghost** [comment]
If I switch off the import of the timeline from Bluesky, this behaviour stops.
It seems to be because Friendica imports its own posts from Bluesky and then forwards them to Twitter a second time.

**Comment by annando** [comment]
I'm totally unsure, how this could happen at all - for several reasons. The "remote self" functionality is also not enabled for Bluesky, so imported posts should never be repeated. Also it shouldn't even happen to have got duplicates on Twitter, since their API checks for duplicated content.

**Comment by annando** [comment]
I don't think that the problem still exists, since we changed a lot for this connector. Since the user deleted their account, they sadly can't confirm the behaviour.
